util/win11_usb_daemon.ps1

```diff
@@ -0,0 +1,27 @@
+$scriptFilePath = $MyInvocation.MyCommand.Path
+$scriptDirPath = (Get-Item $scriptFilePath).DirectoryName
+
+function Confirm-AdministatorPrivledgesActive {
+    return ([Security.Principal.WindowsPrincipal] [Security.Principal.WindowsIdentity]::GetCurrent()).IsInRole([Security.Principal.WindowsBuiltInRole] 'Administrator')
+}
+
+if (!(Confirm-AdministatorPrivledgesActive)) {
+    # user has not deliberately run the script as admin
+    # print text warning and pause so they know a UAC prompt is coming
+    Write-Host "Elevating to administrator runtime. You will prompted via UAC."
+    # Write-Host "Press any *LETTER* key to continue..."
```

Comment:
These two lines look unused?

Reply:
Yeah actual question, I did have it say it was gonna prompt for admin before doing so should we do that? I got annoyed with it mid dev. I think I can delete these.

Reply:
I will delete it